package.json

```diff
@@ -31,7 +31,7 @@
     "@glimmer/util": "^0.26.2"
   },
   "devDependencies": {
-    "@glimmer/application": "^0.5.0",
+    "@glimmer/application": "^0.7.2",
```

Comment:
Yes, it is necessary for these tests to pass. I could probably do this upgrade in a separate PR still, but it would need to be merged first.